src/app/legacy/types.ts

```diff
@@ -11,3 +11,13 @@ export interface LegacyQuote {
 export interface LegacyRedemptionQuoteResult extends QuoteResult {
   legacy: LegacyQuote | null
 }
+
+export interface LegacyToken {
+  name: string
+  symbol: string
+  address: string | undefined
+  decimals: number
+  image: string
+  chainId?: number
+  logoURI?: string
+}
```

Comment:
Adding a new type to decouple from `Token` type from `constants/token`. Will follow up with a PR to refactor quite a bit around `Token`.